style.css

```
@@ -0,0 +1,345 @@
+  gap: 5%;
+}
+
+.main-img:nth-child(2) {
+  text-align: right;
+}
+
+.info{
+  flex: 1;
+}
+
+.img-tag {
+  color: var(--brand-blue);
+  font-size: 18px;
+  line-height: 25px;
+  font-weight: 700;
+  margin-bottom: 12px;
+}
+
+.img-description {
+  font-size: 24px;
+  font-weight: 500;
+  line-height: 120%;
+  letter-spacing: 0.08em;
+  margin-top: 24px;
+}
+
+/* Tablet */
+@media (min-width:768px) and (max-width: 1199px) {
+  .logo{
+  margin-left:24px;
+ }
+ 
+ #login-button{
+  margin-right:24px;
+ }
+
+ .banner {
+  background-size:auto;
+  position:relative;
+  justify-content: center;
+}
+
+ #banner-top.banner{
+  height:771px;
+ }
+
+ #banner-bottom.banner{
+  height:927px;
+ }
+
+ #banner-top .wrapper{
+  position:absolute;
+  top:84px;
+ }
+
+ #banner-bottom .wrapper{
+  position:absolute;
+  top:201px;
+ }
+ 
+ #banner-top .wrapper .button {
+  display: block; 
+  margin: 0 auto;
+  text-align: center;
+  width:100%;
+  max-width:355px;
+  max-height:60px;
+  white-space: nowrap; 
+}
+
+ #banner-top.wrapper,
+ #banner-bottom.wrapper{
+  position: relative;
+  text-align: center;
+ }
+
+ #banner-top .wrapper h1,
+ #banner-bottom .wrapper h1,
+ #banner-top .wrapper .button{
+   margin: 0 auto;
+   text-align: center;
+ }
+
+  #banner-top .wrapper h1{
+    margin-bottom:24px;
+  }
+
+ header{
+  margin-top:10px;
+ }
+
+ #banner-bottom.wrapper{
+  position: absolute;
+  top:201px;
+ }
+
+ .main-img {
+  display:block; 
+}
+
+.main-img img{
+  width:696px;
+  height:524px;
+}
+}
+  
+/* Mobile */
+@media (min-width:375px) and (max-width: 767px) {
+  .logo{
+    margin-left:16px;
+  }
+   
+  .button.pill-button {
+    text-align: center;
+  }
+
+  #login-button{
+    margin-right:16px;
+  }
+
+  .banner {
+    position:relative;
+    justify-content: center;
+  }
+
+  #banner-top{
+    background-size:626px 281px;
+  }
+
+  #banner-top .wrapper{
+    position:absolute;
+    top:48px;
+   }
+
+   #banner-bottom .wrapper{
+    position:absolute;
+    top:121px;
+   }
+
+  #banner-top.banner{
+    height:540px;
+   }
+  
+   #banner-bottom{
+    background-size:497px 269px;
+   }
+
+   #banner-bottom.banner{
+    justify-content: center;
+    text-align: center;
+   }
+
+   #banner-top .wrapper .button {
+    display: block; 
+    margin: 0 auto;
+    text-align: center;
+    width:100%;
+    max-width:154px;
+    max-height:48px;
+    white-space: nowrap; 
+    font-weight:600;
+    font-size:16px;
+    line-height: 19.09px;
+  }
+
+  #banner-top.wrapper,
+  #banner-bottom.wrapper{
+    position: relative;
+    text-align: center;
+  }
+  
+   #banner-top .wrapper h1,
+   #banner-top .wrapper .button{
+    margin: 0 auto; 
+    text-align: center; 
+  }
+
+  #banner-top .wrapper h1{
+    margin-bottom:18px;
+    font-size:32px;
+    line-height:44.8px ;
+  }
+
+   header{
+    margin-top:10px;
+  }
+
+   #banner-bottom.wrapper{
+    position: absolute;
+    top:201px;
+  }
+
+   .main{
+    margin-left:16px;
+    margin-right:16px;
+  }
```

prettier 확장프로그램과 formatOnSave를 설정하시면 기본적인 formatting이 적용 될거에요
```suggestion
  .main {
    margin-left: 16px;
    margin-right: 16px;
  }
```